react-search/my-app/src/Search.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react';
+
+type Prop = {
+  quotes: string[];
+};
+export function Search({ quotes }: Prop) {
+  const [searchTerm, setSearchTerm] = useState('');
+  const [filtered, setFiltered] = useState(quotes);
```

Comment:
This shouldn't be state since it can just be a local variable. Then in `handleInput` all you do is call `setSearchTerm` (remove the code that filters the quotes and move that into a local variable inside the Search function). When the component re-renders it will recalculate the filtered list and use it.